configs/templates/skylab-2.0.0/spack.yaml

```diff
@@ -0,0 +1,93 @@
+spack:
+  concretizer:
+    unify: when_possible
+
+  view: false
+
+  packages:
+    fiat:
+      version: [1.0.0]
+    ectrans:
+      version: [1.0.0]
+
+  view: false
+  include: []
+
+  specs:
+    # Virtual environment packages
+    - base-env@1.0.0
```

Comment:
`1.0.0` or `2.0.0` for virtual environment packages?

Reply:
Good question. It doesn't really matter since those environment packages have no real meaning, and some of the env packages haven't changed at all, some have (i.e. new dependencies). We could probably just strip the version and it be fine, too.

Reply:
Trying this now. Also removing the unneeded env packages base-env and jedi-base-env, because these are implicitly required in the other env packages.